Build/scripts/pgo/pre_pgo.cmd

```diff
@@ -12,6 +12,13 @@
 :: - build (using PGO profile)
 :: - post_pgo.cmd
 
+@echo off
+
+if "%PogoConfig%"=="False" (
+    echo "---- Not a Pogo Config. Skipping step."

```

Comment:
Emm, not just personal preference, the extra double quotes will be put into console output or log file, echo command doesn't interpret them.

---
In reply to: [61511010](https://github.com/Microsoft/ChakraCore/pull/863#discussion_r61511010) [](ancestors = 61511010)